spec/manifest_spec.rb

```diff
@@ -46,7 +47,7 @@
       experiments: {
         hashParams: true
       }
-    }
+    }.dup
```

Comment:
I thought I was seeing the hash get cached between tests. I'm not sure if that was the case or not, but seems pretty harmless